homeassistant/components/volvooncall/lock.py

```diff
@@ -27,8 +29,10 @@ async def async_setup_platform(
 class VolvoLock(VolvoEntity, LockEntity):
     """Represents a car lock."""
 
+    instrument: Lock
+
     @property
-    def is_locked(self):
+    def is_locked(self) -> bool | None:
```

Comment:
I have tracked this to `self.attrs.get("carLocked")` in https://github.com/molobrakos/volvooncall/blob/89f3619ca6c9829c228b62a4a31f16fccbeca732/volvooncall/volvooncall.py#L226-L228

That makes it nullable I think.